hooks/useUserIdentity.ts

```diff
@@ -0,0 +1,93 @@
+import { useEffect, useState } from 'react'
+import { useRouter } from 'next/router'
+import AccountsSDK from '@livechat/accounts-sdk'
+import { Config } from 'lib/config'
+
+type UserIdentity = {
+  account_id: string
+  access_token: string
+  expires_in: number
+  organization_id: string
+  scope: string
+  token_type: 'Bearer'
+}
+
+const accountsSDK = new AccountsSDK({
+  client_id: Config.lcClientId,
+  server_url: Config.lcAccountsURL,
+})
+
+function useUserIdentity() {
+  const [userIdentity, setUserIdentity] = useState<UserIdentity | null>(null)
+  const [loading, setLoading] = useState<boolean>(false)
+  const { asPath, replace } = useRouter()
+
+  useEffect(() => {
+    let identity: UserIdentity = {
+      account_id: '',
+      access_token: '',
+      expires_in: 0,
+      organization_id: '',
+      scope: '',
+      token_type: 'Bearer',
+    }
+
+    if (asPath !== '' && asPath !== '/') {
+      asPath
+        .replace('/#', '')
+        .split('&')
+        .forEach((item) => {
+          const key = item.split('=')[0]
+          const value = item.split('=')[1]
+
+          switch (key) {
+            case 'access_token':
+              identity.access_token = value
+              break
+            case 'expires_in':
+              identity.expires_in = parseInt(value)
+              break
+            case 'scope':
+              identity.scope = value
+              break
+            default:
+              break
+          }
+        })
+
+      setUserIdentity(identity)
+    }
```

Comment:
I would recommend sticking to methods for reading user data provided in the accountsSDK docs, so we wouldn't confuse future users within 2 different flows

https://github.com/livechat/accounts-sdk#flows <- Please check flow for redirects